### DISCUSSION TITLE
Command is not valid

### DISCUSSION THREAD

**Discussion by laughattears** [pyenv-win/pyenv-win-venv#24] [General]
I run this
`pyenv-win-venv install 3.12.1 myenv  `
but got error 
`Command is not valid. Run "pyenv-win-venv help" for the HelpMenu`

**Comment by arzkar** [comment]
Hey, sorry for the late reply. I didn't get any notifications for new discussions for some reason.

There's a bug with the `pyenv-win-venv` command, it's expecting `bat` or `ps1` as the first argument to detect the current shell.
It's a poor design choice that was overlooked because everyone uses `pyenv-venv` which automatically detects the shell. I will be working on a fix for this.

In the meantime, you can use the `pyenv-venv` command instead.

**Comment by arzkar** [comment]
I have pushed an update `v0.6.5` which fixes this issue.
Update the CLI using `pyenv-venv update self`
